### DISCUSSION TITLE
Extra版本怎么才能不OutOfMemeory?

### DISCUSSION THREAD

**Discussion by oszc** [fishaudio/Bert-VITS2#284] [Show and tell]
机器配置为： 
CPU: amd Ryzen 7 5800  
Mem:32GB
Grp: 4090 24GB
训练模型坚持不了几轮？在哪里配置可以降低内存使用？

**Comment by Stardust-minus** [comment]
调一下batch size和dataloader配置。默认是512G内存+80G显存的config，直接用容易爆
